omod/src/main/webapp/resources/scripts/emr.js

```diff
@@ -87,6 +87,37 @@ var emr = (function($) {
             }
         },
 
+        serverValidationErrorMessage: function(response) {
+    		var errorMessage = this.message("uicommons.generalError.message", "An error has occurred");
+        	if (response.data != null && response.data.error != null) {
+        		if (response.data.error.message != null) {
```

Comment:
Is this like one huge if clause? The message being null doesn't mean their are no global errors and field errors